connect/mirror-client/src/main/java/org/apache/kafka/connect/mirror/RemoteClusterUtils.java

```diff
@@ -1,94 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one or more
- * contributor license agreements. See the NOTICE file distributed with
- * this work for additional information regarding copyright ownership.
- * The ASF licenses this file to You under the Apache License, Version 2.0
- * (the "License"); you may not use this file except in compliance with
- * the License. You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.apache.kafka.connect.mirror;
-
-import org.apache.kafka.common.TopicPartition;
-import org.apache.kafka.clients.consumer.OffsetAndMetadata;
-
-import java.util.Map;
-import java.util.Set;
-import java.util.concurrent.TimeoutException;
-import java.time.Duration;
-
-
-/** Convenience methods for multi-cluster environments. Wraps MirrorClient (@see MirrorClient).
- *  <p>
- *  Properties passed to these methods are used to construct internal Admin and Consumer clients.
- *  Sub-configs like "admin.xyz" are also supported. For example:
- *  </p>
- *  <pre>
- *      bootstrap.servers = host1:9092
- *      consumer.client.id = mm2-client
- *  </pre>
- *  <p>
- *  @see MirrorClientConfig for additional properties used by the internal MirrorClient.
- *  </p>
- */
-public final class RemoteClusterUtils {
```

Comment:
I see this isn't currently being used, but we should be careful about removing it completely. This might just be the groundwork for additional testing which hasn't been completed yet. Seems like this class was added as part of the MirrorMaker2 work, which still has at least one open [PR for system tests](https://issues.apache.org/jira/browse/KAFKA-8929) which might rely on this utility class. 

@ryannedolan can you confirm whether this class is definitively no longer needed?

Reply:
This is a public interface and js definitely being used in external projects.

Reply:
Oh thanks, didn't notice it was public. Glad I asked

Reply:
Oh, I see. I will revert it. Thanks @ryannedolan! :smile: